tidy install desktop modeler page (#5799)
camunda/camunda-docs: docs/components/modeler/desktop-modeler/install-the-modeler.md

description: "Learn how to install Camunda Desktop Modeler, a desktop application for modeling BPMN, DMN, and Forms and support building executable diagrams with Camunda."
---

> This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda. Alternatively, you can use our cloud-based [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).

Reviewer: Not sure how much we need to describe the modeler on this page. Do we assume folks visit the page without prior context? If so, consider to shorten to

```suggestion
Desktop Modeler allows you to model BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda, alongside your favorite IDE. This document guides you through Desktop Modeler installation. 

:::tip
If you search for cloud-based modeling instead, try out [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).
:::
```

> This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda. Alternatively, you can use our cloud-based [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).

Reviewer: I'm really not sure why we need the "cloud" hint here, it explicitly is the Desktop Modeler downloads page.

> This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda. Alternatively, you can use our cloud-based [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).

Reviewer 2: Right. We don't send people to Desktop Modeler at https://docs.camunda.io/docs/components/modeler/web-modeler/launch-web-modeler/

> This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda. Alternatively, you can use our cloud-based [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).

Author: Removed 👍

> This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms and supports you in building executable diagrams with Camunda. Alternatively, you may utilize [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md), our cloud-based Modeler.

Reviewer 3: ```suggestion
This document guides you through Desktop Modeler installation, our local modeler. Desktop Modeler is a desktop application for modeling BPMN, DMN, and Forms, and supports you in building executable diagrams with Camunda. Alternatively, you can use our cloud-based [Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md).
```
Suggestion

## Installation

To install [Desktop Modeler](./index.md) for Windows, macOS, and Linux, visit the [Camunda downloads page](https://camunda.com/download/modeler/). Select your preferred version and take the following steps:

1. Unpack the archive (any platform) or open the `.dmg` file (macOS).
2. For macOS users, move the app to your applications folder (macOS).
3. Start the Camunda Modeler (Windows) or `camunda-modeler` (Linux) executable, or the Camunda Modeler application (macOS).

## Wire File Associations

On Windows and Linux you can carry out additional steps to register Modeler as the default editor for BPMN, DMN, RPA, and Form files. With macOS, Modeler is automatically registered as the default editor.

### Windows

> To make Modeler the default editor for `.bpmn`, `.dmn`, `.rpa`, and `.form` files, execute `support/register_fileassoc.bat` in your terminal.

Reviewer 2: You typically run the script via double-click:

```suggestion
To make Modeler the default editor for `.bpmn`, `.dmn`, `.rpa`, and `.form` files, open `support/register_fileassoc.bat`.
```

### Linux

> To create a [desktop file](https://specifications.freedesktop.org/desktop-entry-spec/latest/) and make Modeler the default editor for `.bpmn`, `.dmn`, `.rpa`, and `.form` files, execute `support/xdg_register.sh` in your terminal.

Reviewer: We probably want to change this to 

> "To register the Modeler for `.bpmn`, `.dmn`, ... execute ..."

> To create a [desktop file](https://specifications.freedesktop.org/desktop-entry-spec/latest/) and make Modeler the default editor for `.bpmn`, `.dmn`, `.rpa`, and `.form` files, execute `support/xdg_register.sh` in your terminal.

Reviewer: I'd personally say "from your terminal".

> To create a [desktop file](https://specifications.freedesktop.org/desktop-entry-spec/latest/) and make Modeler the default editor for `.bpmn`, `.dmn`, `.rpa`, and `.form` files, execute `support/xdg_register.sh` in your terminal.

Author: Adjusted.
